test/test.js

```diff
@@ -297,6 +305,14 @@ describe('chai-immutable', function () {
 
   describe('TDD interface', function () {
     describe('equal assertion', function () {
+      it(
+        'fails when only the "expected" value is an Immutable collection',
+        function () {
+          var fn = function () { assert.equal([], List()); };
+          expect(fn).to.throw(Error);
```

Comment:
Argh, sorry, I should have been more explicit on my original comment. Would you mind using `assert.throw` here?

My rationale behind that is to keep the APIs "self contained". It may help future contributors who only deal with one API only, they don't have to use both APIs at the same time. Makes sense?